Servers/UI/OJS.Servers.Ui/ClientApp/src/components/submissions/details/SubmissionDetails.tsx

```diff
@@ -117,47 +118,79 @@ const SubmissionDetails = () => {
         [ currentSubmission, canAccessAdministration ],
     );
 
-    if (isNil(currentSubmission)) {
-        return <div>No details fetched.</div>;
-    }
+    const renderSubmissionDetails = useCallback(
+        () => (
+            <div className={styles.detailsWrapper}>
```

Comment:
This needs to be refactored as its too heavy for a single render

Reply:
Done.